spec/models/personal_details_spec.rb

```diff
@@ -7,7 +7,6 @@
   it { is_expected.to validate_presence_of :date_of_birth }
   it { is_expected.to validate_presence_of :nationality }
 
-  it { is_expected.to validate_presence_of(:phone_number).on(:update) }
-  it { is_expected.to validate_presence_of(:email_address).on(:update) }
-  it { is_expected.to validate_presence_of(:address).on(:update) }
+  # TODO: create relation between PersonalDetails and Candidate
```

Comment:
See comment on `contact_details_spec.rb`

Reply:
+1 on this